### PR TITLE
docs(s3): update Bucket policy doc MTA-3844

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: macros/storage/important-bucket-policy.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+macro: important-bucket-policy
+---
+
+<Message type="important">
+    - You may lose access to your bucket if you are not the [owner](/identity-and-access-management/iam/concepts/#owner) of the organization and if you are not explicitly allowed by the bucket policy.
+    - Each bucket can have only one bucket policy.
+    - Pushing a new bucket policy to a bucket overwrites any existing bucket policy.
+    - The [owner](/identity-and-access-management/iam/concepts/#owner) of the organization always has full rights and access to all resources and features in their Organization.
```
Comment (nerda-codes) [inline macros/storage/important-bucket-policy.mdx:9]:
```suggestion
    - The [owner](/identity-and-access-management/iam/concepts/#owner) of the Organization always has full rights and access to all resources and features in their Organization.
```

File: storage/object/api-cli/create-bucket-policy.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  - You have [created an Object Storage bucket](/storage/object/how-to/create-a-bucket/)
+  - You have [installed the AWS CLI](/storage/object/api-cli/object-storage-aws-cli/)
+  - You have [created an IAM policy](/identity-and-access-management/iam/how-to/create-policy/) to grant access to your users and applications
+</Message>
+
+## How to create a bucket policy
+
+1. Create a `bucket-policy.json` file.
+
+2. Open it in a code editor and paste the code below inside. This statement **ONLY** allows the specified user to see the bucket and its objects.
```
Comment (nerda-codes) [inline storage/object/api-cli/create-bucket-policy.mdx:32]:
```suggestion
2. Open it in a code editor and paste the code below inside. This statement **only allows the specified user** to see the bucket and its objects.
```

File: storage/object/api-cli/create-bucket-policy.mdx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                ],
+                "Resource": [
+                    "<BUCKET_NAME>",
+                    "<BUCKET_NAME>/*"
+                ]
+            }
+        ]
+    }
+    ```
+3. Replace the `<USER_ID>` placeholder with your ID or the one of the user you want to grant access. You can also grant access to an [application](/storage/object/api-cli/bucket-policy/#principal).
```
Comment (nerda-codes) [inline storage/object/api-cli/create-bucket-policy.mdx:56]:
```suggestion
3. Replace the `<USER_ID>` placeholder with the ID of the user to which you want to grant access. You can also grant access to an [application](/storage/object/api-cli/bucket-policy/#principal).
```

File: storage/object/api-cli/create-bucket-policy.mdx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+4. Replace the `<BUCKET_NAME>` placeholders with the name of the concerned bucket. Refer to the [resource documentation](/storage/object/api-cli/bucket-policy/#resource) for more information.
+
+<Message type="note">
+Refer to the [Bucket policies description](/storage/object/api-cli/bucket-policy/#bucket-policy-description) for more details on each string.
+</Message>
+
+## How to apply a bucket policy
+
+Make sure that you have [installed the AWS CLI](/storage/object/api-cli/object-storage-aws-cli/) before continuing.
```
Comment (nerda-codes) [inline storage/object/api-cli/create-bucket-policy.mdx:65]:
```suggestion
Make sure that you have [installed the AWS CLI](/storage/object/api-cli/object-storage-aws-cli/) before proceeding.
```

File: storage/object/concepts.mdx [hunk 5]
```diff
@@ -165,6 +167,14 @@ Creating a [lifecycle rule](/storage/object/how-to/manage-lifecycle-rules/) base
 
 <Concept>
 
+## principal
+
+A principal is the target of a [bucket policy](#bucket-policy). They acquire the rights and permissions defined in the policy. The principal of a bucket policy can be an IAM user, or an IAM application. Bucket policies can have several principals attached to it.
```
Comment (nerda-codes) [inline storage/object/concepts.mdx:172]:
```suggestion
A principal is the target of a [bucket policy](#bucket-policy) and acquires the rights and permissions defined in the policy. The principal of a bucket policy can be an IAM user, or an IAM application. Bucket policies can have several principals attached to it.
```

**Review by nerda-codes — APPROVED** [review]

**Comment by SamyOubouaziz** [comment]
waiting for @crlptl to approve before merge

File: storage/object/api-cli/combining-iam-and-object-storage.mdx [hunk 6]
```diff
@@ -1,280 +1,149 @@
...
-    - When you create a Scaleway API key with IAM, you are prompted to specify the API key’s [preferred Project for Object Storage](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/). This API key will always use this Project when carrying out Object Storage actions via any API/CLI.
-    - The [preferred Project](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/) must always be the one of the bucket where you wish to perform an action.
-</Message>
+### IAM
 
-<Message type="note">
-  Before the [arrival of IAM](/identity-and-access-management/iam/reference-content/migration/) principals in a Scaleway bucket policy were defined by the `project_id`. Now they can either be specified by the `user_id` or the `application_id`.
-</Message>
+[Identity and Access Management (IAM)](/identity-and-access-management/iam/concepts/#iam) allows you to control and manage access to your resources. For users of Scaleway Object Storage, this means that besides configuring access via bucket policies, you can use IAM policies to configure permissions.
```
Comment (crlptl) [inline storage/object/api-cli/combining-iam-and-object-storage.mdx:29]: We could say again here "at the project level". Even though it's written below, that's the main fact

File: storage/object/api-cli/combining-iam-and-object-storage.mdx [hunk 7]
```diff
@@ -1,280 +1,149 @@
...
+        "Statement": [
+            {
+                "Sid": "Allow application to GET objects",
+                "Effect": "Allow",
+                "Principal": "application-id-1d96-0b4c-54cd",
+                "Action": [
+                    "s3:GetObject",
+                ],
+                "Resource": [
+                    "bucket-A",
```
Comment (crlptl) [inline storage/object/api-cli/combining-iam-and-object-storage.mdx:105]: the first resource is useless here in this second statement

File: storage/object/api-cli/create-bucket-policy.mdx [hunk 8]
```diff
@@ -0,0 +1,106 @@
+---
+meta:
+  title: Creating and applying a bucket policy
+  description: This page shows how to create and apply a bucket policy to a specific bucket using the Scaleway CLI
```
Comment (ldecarvalho-doc) [inline storage/object/api-cli/create-bucket-policy.mdx:4]: it's the AWS CLi, no?

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**